src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,106 @@
+- DB Console cookies are marked `Secure` for the browser when the cluster is running in secure mode. [#119262][#119262]
+- The DB Console `session` cookie is now marked `HttpOnly` to prevent it from being read by any Javascript code. [#119262][#119262]
+
+<h3 id="v23-1-17-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting](https://cockroachlabs.com/docs/v23.1/cluster-settings) `server.max_open_transactions_per_gateway`, when set to a non-negative value, prevents users without the `admin` role from executing a query if more than this number of transactions are open on the current gateway node. [#118949][#118949]
+- Added support for [index hinting](https://www.cockroachlabs.com/docs/v23.1/table-expressions#force-index-selection) with `INSERT` and `UPSERT` statements. This allows `INSERT ... ON CONFLICT` and `UPSERT` queries to use index hints in the same way as `UPDATE` and `DELETE` statements. [#119600][#119600]
+
+<h3 id="v23-1-17-operational-changes">Operational changes</h3>
+
+- You can now pass the `--include-range-info` flag when [generating a diagnostic bundle](https://www.cockroachlabs.com/docs/v23.1/cockroach-debug-zip) to include problem ranges. [#119233][#119233]
+- In unredacted diagnostic bundles, two columns have been added to the `crdb_internal.transaction_contention_events` table:
+    - `waiting_stmt_query`: the query of the statement that is waiting to run.
+    - `blocking_txn_queries_unordered`: an unordered list of the blocking transaction's queries. [#119422]
+
+  [#119422]
+
+<h3 id="v23-1-17-db-console-changes">DB Console changes</h3>
+
+- The [**Statements** page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page) now always shows the entire selected period, instead of only the period that contains data. [#118811][#118811]
+- The [**Overload Dashboard**](https://www.cockroachlabs.com/docs/v23.1/ui-overload-dashboard) page now includes two additional graphs:
+    - **Elastic CPU Utilization**: Shows the actual CPU used for elastic work compared with the configured limit.
+    - **Elastic CPU Exhausted Duration Per Second**: Shows how much time, in milliseconds, that elastic work has been subject to CPU exhaustion. [#118908][#118908]
+
+<h3 id="v23-1-17-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where creating a [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview) that targeted tables with a zero-scale `DECIMAL(n)` column, `format='avro'`, and `diff` would cause a panic. [#118848][#118848]
+- - Fixed a bug where a [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview.html) could omit events in rare cases, logging the error `cdc ux violation: detected timestamp ... that is less or equal to the local frontier`. This could happen in the following scenario:
```

Comment:
```suggestion
- Fixed a bug where a [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview.html) could omit events in rare cases, logging the error `cdc ux violation: detected timestamp ... that is less or equal to the local frontier`. This could happen in the following scenario:
```